roadmap.rst

```diff
@@ -15,8 +15,67 @@ and the PSF in early Jan 2016):
 
    proposal_december2015
 
-Current status:
+Status update June '16
+----------------------
 
-- phase 1 of the proposal has been completed (March 2016)
-- work on phase 2 is in progress, see the
-  `Milestone on Github <https://github.com/mingwpy/mingwpy/milestones/Phase%202>`_
+**Progress on milestones in the proposal**
+
+Phase 1 of the proposal has been finished.  The compiler toolchain has now been
+re-built by several people based on the public build scripts and documentation.
```

Comment:
links to `public build script` and `documentation` would be nice to see, as well as prooflinks for re-built versions (just my perfectionism)